libs/state/effects/src/lib/effects.service.ts

```diff
@@ -223,15 +235,7 @@ export class RxEffects implements OnDestroy, OnDestroy$ {
     return <V>(source: Observable<V>) =>
       source.pipe(
         untilDestroyed(this),
```

Comment:
nit: if you rely on `Destroy` ref, you could as well replace `untilDestroyed(this)` with the framework provided operator. (Maybe in a subsequent PR)

Reply:
I also thought about this already. Will open another PR if this one get's merged!
Thanks for pointing this out.